app/components/Events.vue

```diff
@@ -1,21 +1,24 @@
 <template>
-    <StackLayout>
-        <StackLayout class="eventItem" v-for="e in events" >
-            <Label class="category" :text="e.category" width="100%"/>
-            <Label class="title" :text="e.title" width="100%"/>
-            <Label class="text description" :text="e.description" width="100%"/>
-            <Label class="spacer" />
-            <StackLayout  class="hbox" orientation="horizontal" >
-                <Label class="date textLeft" width="45%">D: {{ e.date }}</Label>
-                <Label class="location textRight" width="55%">L: {{ e.location}}</Label>
-            </StackLayout>
-            <Label class="text time" width="100%">Start: {{ moment( e.start_time ) }}</Label>
-            <StackLayout class="text hbox " orientation="horizontal">
-                <Label class="time textLeft" width="45%">End: {{ moment( e.end_time ) }}</Label>
-                <Label class="people textRight" width="55%">P: {{ e.people }}</Label>
+    <ScrollView>
+        <StackLayout class="eventContainer" >
+            <StackLayout class="eventItem" v-for="e in events" >
+                <Label class="category" :text="e.category" width="100%"/>
+                <Label class="title" :text="e.title" width="100%"/>
+                <Label class="text description" :text="e.description" width="100%"/>
+                <Label class="spacer" />
+                <StackLayout  class="hbox" orientation="horizontal" >
+                    <Label class="date textLeft" width="45%">D: {{ e.date }}</Label>
+                    <Label class="location textRight" width="55%">L: {{ e.location}}</Label>
+                </StackLayout>
+                <Label class="text time" width="100%">Start: {{ moment( e.start_time ) }}</Label>
+                <StackLayout class="text hbox " orientation="horizontal">
+                    <Label class="time textLeft" width="45%">End: {{ moment( e.end_time ) }}</Label>
+                    <Label class="people textRight" width="55%">P: {{ e.people }}</Label>
+                </StackLayout>
             </StackLayout>
+            <Label height="20px" />
```

Comment:
There must be a better way to do this than having a blank label lol. I'll leave another comment if I find anything to use instead.

Reply:
Industry standard.